assets/css/translation.css

```diff
@@ -126,3 +126,116 @@ html[dir='rtl'] #srcLanguages {
     padding: 0 15px;
     position: relative;
 }
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#applyingWidth {
+    width: 35%;
+}
+
+#applyingPadding {
+    padding: 5px 5px 0 0;
+}
+
+#translateDoc {
+    margin-top: 1em;
+}
+
+#fileDropBackdrop {
+    z-index: 1000;
+}
+
+#dropDocumentDiv {
```

Comment:
Let's call it `dropDocumentWell` like the `.well` class. As I mentioned earlier, we should have element names in our ids.

Reply:
Did not see this.Thanks for pointing out.